tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -29,6 +29,26 @@ describe('main_html5 wrapper tests', function () {
 
   beforeEach(function() {
     vtc = new mock_vtc();
+    vtc.interface.EVENTS.ASSET_DIMENSION = "assetDimension";
```

Comment:
Would it be better to define these inside `mock_vtc.js`?

Reply:
Yea, these are also defined in each test that requires them below. I'll fix these.

Reply:
Fixed